domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -45,11 +45,11 @@
 
   public static final Map<String, String> MODULE_SPECIFIC_ARGS  = new HashMap<>(); //NOSONAR
 
-  private static final int FORM_ATTRIBUTE_SIZE_MAX = 32768;
   private static final String       HTTP_PORT_SETTING               = "http.port";
   private static final Logger       log                             = LogManager.getLogger(RestVerticle.class);
   private static String             deploymentId                     = "";
 
+  private static HttpServerOptions httpServerOptions = new HttpServerOptions();
```

Comment:
```suggestion
  // default options for server; may be changed in init hook
  private static HttpServerOptions httpServerOptions = new HttpServerOptions().setCompressionSupported(true);
```
A module might want to change it while processing config parameters before the init hook is called.

Reply:
right. yes.